tensorboard/backend/event_processing/data_provider.py

```diff
@@ -394,7 +393,7 @@ def _decode_blob_key(key):
     return (experiment_id, plugin_name, run, tag, step, index)
 
 
-def _convert_scalar_event(event):
+def _convert_scalar_event(experiment_id, plugin_name, run, tag, event):
```

Comment:
Optional, but IMO a little nicer if we make `event` the first argument for all of these so that the other 4 arguments are more semantically positioned to be optional arguments, even if we don't bother actually making them optional.  Might also be nice to add a `del experiment_id, plugin_name, run, tag  # unused` just to make it clearer that these first two converters ignore all of those arguments.

Reply:
Removed arguments entirely.